app/models/incomeSourceDetails/UIJourneySessionData.scala

```diff
@@ -81,41 +82,48 @@ object AddIncomeSourceData {
   val dateStartedField = "dateStarted"
   val accountingPeriodStartDateField = "accountingPeriodStartDate"
   val accountingPeriodEndDateField = "accountingPeriodEndDate"
-  val createdIncomeSourceIdField: String = "createdIncomeSourceId"
+  val incomeSourceIdField: String = "incomeSourceId"
   val addressField: String = "address"
   val countryCodeField: String = "countryCode"
   val incomeSourcesAccountingMethodField: String = "incomeSourcesAccountingMethod"
-  val hasBeenAddedField: String = "hasBeenAdded"
+  val journeyIsCompleteField: String = "journeyIsComplete"
+  val incomeSourceAddedField: String = "incomeSourceAdded"
 
   def getJSONKeyPath(name: String): String = s"addIncomeSourceData.$name"
 
   implicit val format: OFormat[AddIncomeSourceData] = Json.format[AddIncomeSourceData]
 }
 
-case class ManageIncomeSourceData(
-                                   incomeSourceId: Option[String] = None
-                                 )
+case class ManageIncomeSourceData(incomeSourceId: Option[String] = None,
+                                  reportingMethod: Option[String] = None,
+                                  taxYear: Option[String] = None,
+                                  journeyIsComplete: Option[Boolean] = None)
 
 object ManageIncomeSourceData {
 
   val incomeSourceIdField = "incomeSourceId"
+  val reportingMethodField: String = "reportingMethod"
```

Comment:
I don't see we are actually using:
```
  val reportingMethodField: String = "reportingMethod"
  val taxYearField: String = "taxYear"
```

Reply:
These were in Ed's changes for Manage, now that main has been merged they are in use